featuretools/primitives/standard/binary_transform.py

```diff
@@ -308,6 +310,9 @@ class DivideNumeric(TransformPrimitive):
     input_types = [Numeric, Numeric]
```

Comment:
Please add a docstring for this primitive class too. It can be siimilar to the subtract example I gave